src/main/java/uk/gov/hmcts/reform/userprofileapi/infrastructure/config/SecurityConfiguration.java

```diff
@@ -48,19 +48,28 @@ public void configure(WebSecurity web) {
     @Override
     protected void configure(HttpSecurity http) throws Exception {
 
-        AuthCheckerServiceOnlyFilter authCheckerServiceOnlyFilter =
-            new AuthCheckerServiceOnlyFilter(serviceRequestAuthorizer);
+        AuthCheckerServiceOnlyFilter authCheckerServiceOnlyFilter = new AuthCheckerServiceOnlyFilter(
+                serviceRequestAuthorizer);
 
         authCheckerServiceOnlyFilter.setAuthenticationManager(authenticationManager);
 
-        http
-            .addFilter(authCheckerServiceOnlyFilter)
-            .sessionManagement().sessionCreationPolicy(STATELESS)
-            .and()
-            .csrf().disable()
-            .formLogin().disable()
-            .logout().disable()
-            .authorizeRequests().anyRequest().authenticated()
-        ;
+        http.authorizeRequests()
+                .antMatchers("/actuator/**")
+                .permitAll()
+                .and()
+                .sessionManagement()
+                .sessionCreationPolicy(STATELESS)
+                .and()
+                .csrf()
+                .disable()
+                .formLogin()
+                .disable()
+                .logout()
+                .disable()
+                /*.authorizeRequests()
```

Comment:
Why is this commented out?

Reply:
Since currently we are not enforcing S2S authentication on user profile